PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,38 +1,56 @@
-<!-- Thanks for contributing to the openHAB project! -->
-<!-- Please describe the goal and effect of your PR here. -->
-<!-- Pay attention to the below notes and to *the guidelines* for this repository. -->
-<!-- Feel free to delete any comment lines in the template (starting with "<!--"). -->
-
-<!-- TITLE -->
-
-<!-- Please provide a PR summary in the *Title* above, according to the following schema: -->
-<!-- - If related to one specific addon: Mention the addon shortname in square brackets -->
-<!--   e.g. "[exec]", "[netatmo]" or "[tesla]" -->
-<!-- - If the PR is work in progress: Add "[WIP]" -->
-<!-- - Give a short meaningful description in imperative mood -->
-<!--   e.g. "Add support for device XYZ" or "Fix wrongly handled exception" -->
-<!--   for a new add-on/binding: "Initial contribution" -->
-<!-- Examples: -->
-<!-- - "[homematic] Improve communication with weak signal devices" -->
-<!-- - "[timemachine][WIP] Initial contribution" -->
-<!-- - "Update contribution guidelines on new signing rules" -->
-
-<!-- DESCRIPTION -->
-
-<!-- Please give a few sentences describing the overall goals of the pull request. -->
-<!-- Give enough details to make the improvement and changes of the PR understandable  -->
-<!-- to both developers and tech-savy users. -->
-
-<!-- Please keep the following in mind: -->
-<!-- - What is the classification of the PR, e.g. Bugfix, Improvement, Novel Addition, ... ? -->
-<!-- - Did you describe the PRs motivation and goal? -->
-<!-- - Did you provide a link to any prior discussion, e.g. an issue or community forum thread? -->
-<!-- - Did you describe new features for the end user? -->
-<!-- - Did you describe any noteworthy changes in usage for the end user? --> 
-<!-- - Was the documentation updated accordingly, e.g. the addon README? -->
-<!-- - Does your contribution follow the coding guidelines: -->
-<!--   http://docs.openhab.org/developers/development/guidelines.html -->
-<!-- - Did you check for any (relevant) issues from the static code analysis: -->
-<!--   http://docs.openhab.org/developers/development/bindings.html#static-code-analysis -->
-<!-- - Did you sign-off your work: -->
-<!--   http://docs.openhab.org/developers/contributing/contributing#sign-your-work -->
+<!--
```

Comment:
The benefit of the old style (with the begin/end of line comments) was that you could interleave your text with the comments for a better overview, if you do that now, your text will become invisible. However I think it is more important for the issue template, we use real headers interleaved with the comments there.